bigbluebutton-html5/imports/ui/components/breakout-room/create-breakout-room/room-managment-state/component.tsx

```
@@ -65,128 +65,130 @@ const RoomManagmentState: React.FC<RoomManagmentStateProps> = ({
+        fromRoomId: runningRooms?.find((r) => r.sequence === fromRoom)?.breakoutRoomId,
+      },
+    };
     setMovementRegistered(updatedMovementRegistered);
-    setRooms(updatedRooms);
+    setMoveRegisterRef(updatedMovementRegistered);
+  };
+
+  const moveUser = (userId: string, from: number, to: number) => {
+    const user = userAssignedRooms[userId];
+
+    const userIsOnFromRoom = user && user.includes(from);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)